pkgs/development/libraries/mesa/default.nix

```diff
@@ -176,7 +186,11 @@ stdenv.mkDerivation {
     # - for a cross build (needNativeCLC = true), we provide mesa with `*-clc`
     #   binaries, so it skips building & installing any new CLC files.
     "cross_tools"
-  ];
+  ]
+  # the Dozen drivers depend on libspirv2dxil, but link it statically, and
+  # libspirv2dxil itself is pretty chonky, so relocate it to its own output in
+  # case anything wants to use it at some point
+  ++ lib.optional withDozen "spirv2dxil";
```

Comment:
AFAICT, this change only affects whether there’s a `spirv2dxil` output, not whether anything is actually built?

Reply:
If I understand it correctly, it only gets built if the d3d12 gallium driver is enabled.
So this is only for making sure we don't have an empty output.
(But the dependencies for it could still be conditonalized)